shells/chrome/src/panel.js

```diff
@@ -103,7 +103,9 @@ var node = document.getElementById('container');
 function reload() {
   setTimeout(() => {
     ReactDOM.unmountComponentAtNode(node);
-    node.innerHTML = '';
+    if (node) {
```

Comment:
`node` should always exist here, especially since we're about to call `ReactDOM.render` with it. This `if` here is confusing for a reader because it seems like it should be impossible; it's not clear that you're just checking to make flow happy. It might be less confusing to have an early throw if `node` doesn't exist, if you're just trying to satisfy flow.